src/tests/CreateCarForm.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Provider } from 'react-redux'; // Import Provider
+import store from '../redux/store';
+import '@testing-library/jest-dom/extend-expect';
+import CarForm from '../components/car/CreateCarForm';
+
+describe('CarForm component', () => {
+  test('matches snapshot', () => {
+    const { asFragment } = render(
+      <Provider store={store}>
+        <CarForm />
+      </Provider>,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test('form submission', async () => {
+    const { getByLabelText, getByText, findByText } = render(
+      <Provider store={store}>
+        <CarForm />
+      </Provider>,
+    );
+    const nameInput = getByLabelText('Name');
+    const makeInput = getByLabelText('Make');
+    const modelInput = getByLabelText('Model');
+    const yearInput = getByLabelText('Year');
+    const submitButton = getByText('Create Car');
```

Comment:
- [ ] This test is failing:

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/264bde2c-1eca-4739-92b5-82315a1dbfa4)

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/a2c6175d-e91b-4f22-9aed-cc7b0a3fb76d)

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/d8714972-7dd9-4564-b10d-6d2fc76e1fff)

This is because the label is not 'Name', but it is 'Name:'.  Notice the colon.

Same for Make, Model, and Year.

Kindly, fix this test so it passes. 👍